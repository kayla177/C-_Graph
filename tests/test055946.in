ENTITY EJTROTCY name_EJTR journal
ENTITY 13BZTLKH name_13BZ journal
ENTITY SJYRR2SG name_SJYR author
RELATIONSHIP EJTROTCY collaborated_with 13BZTLKH 4.8
PRINT EJTROTCY
HIGHEST
EXIT
