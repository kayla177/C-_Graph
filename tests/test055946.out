success
success
success
success
13BZTLKH
EJTROTCY 13BZTLKH 4.8
